src/netbase.cpp

```diff
@@ -402,19 +409,22 @@ bool Socks5(const std::string& strDest, uint16_t port, const ProxyCredentials* a
                  * error message. */
                 return false;
             } else {
-                return error("Error while reading proxy response");
+                LogError("Error while reading proxy response\n");
+                return false;
             }
         }
         if (pchRet2[0] != SOCKSVersion::SOCKS5) {
-            return error("Proxy failed to accept request");
+            LogError("Proxy failed to accept request\n");
+            return false;
         }
         if (pchRet2[1] != SOCKS5Reply::SUCCEEDED) {
             // Failures to connect to a peer that are not proxy errors
             LogPrintf("Socks5() connect to %s:%d failed: %s\n", strDest, port, Socks5ErrorString(pchRet2[1]));
```

Comment:
nit - Also noticed this when reviewing the changes. Since we're already changing the file - should Line 413/422 be a `LogError` not a `LogPrintf`?